generate-requirements-full.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+pip install -r requirements-core.txt
+pip install piptools
```

Comment:
`piptools` doesn't exist (and would crash this command!)

```suggestion
pip install pip-tools
```